dev/tools/test/update_engine_version_test.dart

```diff
@@ -19,47 +20,57 @@ void main() {
   late Directory tmpDir;
   late _FlutterRootUnderTest testRoot;
   late Map<String, String> environment;
-  late ProcessRunner processRunner;
+  // late ProcessRunner processRunner;
+
+  io.ProcessResult run(executable, args) {
+    print('Running "$executable ${args.join(" ")}"');
+    final io.ProcessResult result = io.Process.runSync(
+      executable,
+      args,
+      environment: environment,
+      workingDirectory: testRoot.root.absolute.path,
+      includeParentEnvironment: false,
+    );
+    if (result.exitCode != 0) {
+      print('exitCode: ${result.exitCode}');
+    }
+    if (result.stdout.trim().isNotEmpty) {
+      result.stdout.trim().split(io.Platform.lineTerminator).forEach((s) {
+        print('stdout:>$s<');
+      });
+    }
+    if (result.stderr.trim().isNotEmpty) {
+      result.stderr.trim().split(io.Platform.lineTerminator).forEach((s) {
+        print('stderr:>$s<');
+      });
+    }
+    return result;
+  }
 
   setUp(() async {
     tmpDir = localFs.systemTempDirectory.createTempSync('update_engine_version_test.');
     testRoot = _FlutterRootUnderTest.fromPath(tmpDir.childDirectory('flutter').path);
 
     environment = <String, String>{};
-    processRunner = ProcessRunner(
-      defaultWorkingDirectory: testRoot.root,
-      environment: environment,
-      printOutputDefault: true,
-    );
+    environment.addAll(io.Platform.environment);
```

Comment:
I think you _didn't_ want this?

Reply:
> I thought you _didn't_ want this?

I construct environment manually: I use parent's one, but then remove `FLUTTER_PREBUILT_ENGINE_VERSION` explicitly below.
Running with completely blank environment fails to run powershell scripts for some reason.